app-torcx/docker/docker-17.05.ebuild

```diff
@@ -12,7 +12,7 @@ KEYWORDS="amd64 arm64"
 # Explicitly list all packages that will be built into the image.
 RDEPEND="
 	=app-emulation/docker-17.05.0
-	=app-emulation/containerd-0.2.7_p2
+	=app-emulation/containerd-0.2.7_p2-r1
```

Comment:
I think it _should_ but it looks like since this is getting built for torcx and since the torcx packages seem to be built every `build_packages` run it wasn't causing issues.